src/test/java/org/prebid/server/handler/VtrackHandlerTest.java

```diff
@@ -129,7 +150,10 @@ public void shouldRespondWithBadRequestWhenBidIdIsMissing() throws JsonProcessin
         // given
         given(routingContext.getBody())
                 .willReturn(givenVtrackRequest(identity()));
-
+        given(httpResponse.putHeader(HttpUtil.CONTENT_TYPE_HEADER, HttpHeaderValues.APPLICATION_JSON))
```

Comment:
Why you check this here?
Test is about not parsable body.

Consider to change all below tests.